curriculum/challenges/english/16-the-odin-project/top-learn-data-types-and-conditionals/learn-data-types-and-conditionals-question-e.md

```diff
@@ -0,0 +1,45 @@
+---
+id: 65e97260484dd50f720e6fea
+title: Learn Data Types and Conditionals Question E
+challengeType: 15
+dashedName: learn-data-types-and-conditionals-question-e
+---
+# --description--
+
+Strings defined using backticks are called template literals. They are a new way to define strings in JavaScript. They allow you to embed expressions within the string. This is done by wrapping the expression in `${}`.
+
+For example, the following code:
+
+```javascript
+let name = "John";
+let age = 25;
+let greeting = `Hello, my name is ${name} and I am ${age} years old.`;
+```
+
+will result in the `greeting` variable containing the string "Hello, my name is John and I am 25 years old."
+
+# --question--
+
+## --text--
+
+Which of the following is the correct way to define a template literal in JavaScript?
+
+## --answers--
+
+"Hello, my name is ${name} and I am ${age} years old."
```

Comment:
```suggestion
``` `Hello, my name is ${name} and I am ${age} years old.` ```
```

This will display the single backticks, and put the code in codeblocks.